### DISCUSSION TITLE
Select empty set of type

### DISCUSSION THREAD

**Discussion by Fogapod** [geldata/gel#1587] [Q&A] [answered]
Is there a way to select empty set of given type?
I tried these:
- `select <User>{};`: selects all users
- `select <array<User>>{};`: results in https://github.com/edgedb/edgedb/issues/1586

**Comment by elprans** [comment, marked as answer]
`select <User>{};` is the way.  It's a regresion (filed #1589).  You can work around as `SELECT User FILTER false` for now.
